### PR TITLE
Change sign up form for silicon valley to the official one!

### PR DESCRIPTION
**Pull request by AdamEXu**

### CODE REVIEW

**Comment by AdamEXu** [comment]
fixed something rq 😅, glad i caught it before it got merged

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/silicon-valley**](https://counterspell-qkzzb55uo.hackclub.dev/silicon-valley) |
|:-:|
|<a href="https://counterspell-qkzzb55uo.hackclub.dev/silicon-valley">Screenshot of /silicon-valley</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/suh_tBogJiitc1gI-uO8UQ46H0c1jnglJiRputLt7DfEx8qM56m_UMISoJgFSr8JOfk8uAAbediZF2S1KHU_eg.png">(view full size)</a> |







Commit 6fa3655b4813d7cd95bfa73b360ea052b310c1c8 (https://counterspell-qkzzb55uo.hackclub.dev).

File: components/SiliconValley/Locations.js [hunk 1]
```diff
@@ -17,69 +16,21 @@ export default function Locations() {
...
-        </form>
-      </div>
-      <br></br>
-
+    <div className="relative px-6 py-24 text-center neuebit bg-darker retro">
       <p className="text-3xl uppercase">
-        {/* TODO: Replace `South Bay` with your city */}
-        Not in the Bay Area?
+        {/* TODO: Replace `Example City` with your city */}
+        Can't make it to Boston?
```
Comment (samdev-7) [inline components/SiliconValley/Locations.js:23]: Boston?

**Review by samdev-7 — CHANGES_REQUESTED** [review]

**Comment by AdamEXu** [comment]
fixed

**Review by samdev-7 — APPROVED** [review]

**Pull request merged**